package_manifest.yaml

```diff
@@ -28,15 +28,15 @@ foreman_scl_packages:
       - el7-epel
       - el7-extras
       - el7-scl
-      - el7-foreman-rails-nightly
-      - el7-foreman-nightly
+      - el7-foreman-rails-1.22
+      - el7-foreman-1.22
   hosts:
     foreman:
       releasers:
         - "{{ nightly_releaser }}"
       build_package_tito_releaser_args: "{{ nightly_package_tito_releaser_args }}"
       nightly_package_tito_releaser_args:
-        - "--arg jenkins_job=foreman-develop-source-release"
+        - "--arg jenkins_job=foreman-1.22-source-release"
```

Comment:
these jobs don't exist yet, @ehelms is that on your todo?

Reply:
This has the old naming convention: `test_1_22_stable`

Reply:
"oh"

Reply:
I'm not against introducing the new name, but we missed this in the 1.22 cycle.

Reply:
Yea, I was targeting this new convention for 1.23+

Reply:
However, in a release branch, there is no 'nightly' concept right? The source should be from the official hosting site.

Reply:
It's a bit undefined. In the past few releases I updated them and removed them once we had RC1 tarballs.

Reply:
Correction: I kept it in, but changed the releaser once we had tarballs. That means in theory we can do nightly scratch builds of the next RC, but in practice we don't really have this set up.

Reply:
Yeah, I've been following what's in the 1.21 branch

Reply:
So, it seems there are no `test_develop` or `test_1_22` jobs in Jenkins (this is what 1.21 points to).

@ekohl you suggested https://ci.theforeman.org/job/test_1_22_stable, but that does not have artifacts?

Reply:
Every single item in the matrix has an archive. The job picks one of these. This may not be pretty, but it's how it's worked for a long time.

Reply:
"oh²"

Reply:
all updated now